Product Association article update (#596)
akeneo/pim-helpcenter: content/md/products-super-power/product-associations.md

1.  Click on the `Save` button

![Remove associations](../img/Products_ProductsRemoveAssociation.png)

:::info

> A product that has been associated with the 2-way association will be removed from both ways. Please read our [Association Types documentation](manage-your-association-types.html#Overview) for more information.

Reviewer: ```suggestion
A product that has been associated with the 2-way association will be removed from both ways. Please read our [Association Types](manage-your-association-types.html#Overview) article for more information.
```

> A product that has been associated with the 2-way association will be removed from both ways. Please read our [Association Types documentation](manage-your-association-types.html#Overview) for more information.

Reviewer 2: ```suggestion
A product that has been associated using a 2-way association will be removed on both sides. Please read our [Association Types documentation](manage-your-association-types.html#Overview) for more information.
```

Author: OK, new commit done!
